main.rb

```diff
@@ -0,0 +1,576 @@
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
```

Comment:
__[development][rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)